nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/pause/SwitchableRecordProcessorBlocker.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.aws.kinesis.stream.pause;
+
+import java.util.concurrent.CountDownLatch;
+
+public class SwitchableRecordProcessorBlocker implements RecordProcessorBlocker {
```

Comment:
As this is the regular implementation, recommend using `Standard` for the name:
```suggestion
public class StandardRecordProcessorBlocker implements RecordProcessorBlocker {
```

Reply:
I would argue that standard does not provide much information on what behavior the class has

Reply:
Anyway, changed

Reply:
I agree that `Standard` doesn't say anything about what it does, but I think it is sufficient in this case as `Switchable` doesn't make it clear what can be switched. Thanks for making the adjustment.